### PR TITLE
Create dummy/test page

### PR DESCRIPTION
**Pull request by hungryfat92**
An event added to test my syntax of the event page. @shgysk8zer0

### CODE REVIEW

**Comment by shgysk8zer0** [comment]
Could you add a preview build link as a comment?

**Comment by hungryfat92** [comment]
[Preview})https://deploy-preview-52--youthful-dijkstra-321f5b.netlify.com/events/2020-test-for-syntax/)

File: _events/2020-test-for-syntax.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- events
+- kern valley
+- test page
+- Lake Isabella
+offers:
+  - price: 100.50
+    name: Gun Show
+    currency: USD
+organizer:
+  '@type': 'Organization'
```
Comment (shgysk8zer0) [inline _events/2020-test-for-syntax.md]: This should be `'@type': 'Person'`

**Comment by shgysk8zer0** [comment]
You get the idea but need some practice. Closing the issue since you have actual events to work on and this dummy event will never be published (that'd be bad).

**Pull request closed without merging**